components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/listeners/UserManagementAuditLogger.java

```diff
@@ -45,10 +42,18 @@
     private static final Log audit = CarbonConstants.AUDIT_LOG;
     private static final String SUCCESS = "Success";
     private static final String IN_PROGRESS = "In-Progress";
+    public static final String USER_AGENT_QUERY_KEY = "User-Agent";
+    public static final String USER_AGENT_KEY = "User Agent";
+    public static final String REMOTE_ADDRESS_QUERY_KEY = "remoteAddress";
+    public static final String REMOTE_ADDRESS_KEY = "RemoteAddress";
+    public static final String SERVICE_PROVIDER_KEY = "ServiceProviderName";
+    public static final String SERVICE_PROVIDER_QUERY_KEY = "serviceProvider";
+    private final String USER_NAME_KEY = "UserName";
+    private final String USER_NAME_QUERY_KEY = "userName";
 
     @Override
     public boolean doPostAddUser(String userName, Object credential, String[] roleList, Map<String, String> claims,
-            String profile, UserStoreManager userStoreManager) {
+                                 String profile, UserStoreManager userStoreManager) {
```

Comment:
Check the indentation here

Reply:
Indentation seems to be fine on referred with the Check styles.